content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## User facing changes
+- ***Added*** experimental support for the [Kubernetes Service APIs](https://github.com/kubernetes-sigs/service-apis).
+- ***Added*** the new [Workload Entry](https://preliminary.istio.io/docs/reference/config/networking/workload-entry/) resource. This will allow easier configuration for non-Kubernetes workloads to join the mesh.
+- ***Added*** support for configuring Gateway topology settings like number of trusted proxies deployed in front
+- ***Added*** experimental mesh-wide tracing configuration API. This API provides control of trace sampling rates, the maximum tag lengths for URL tags, and custom tags extraction for all traces within the mesh. 
+- ***Added*** experimental request classification filter support. This feature enables operators to configure new attributes for use in telemetry, based on request information. A primary use case for this feature is labeling of traffic by API method.  TODO: doc
+- ***Added*** Wasm extension distribution support. TODO: docs
+- TODO: add a statement around networking v1alpha3 API. Will it be removed in 1.6 or announce deprecation?
+
+## Improvements
+- ***Improved*** support for [Kubernetes Ingress](https://preliminary.istio.io/docs/tasks/traffic-management/ingress/kubernetes-ingress/), adding support for reading certificates from Secrets, `pathType`, and IngressClass.
+- ***Added*** a new `proxy.istio.io/config` annotation to override proxy configuration per pod.
+- ***Added*** support for using appProtocol to select the [protocol for a port](https://preliminary.istio.io/docs/ops/configuration/traffic-management/protocol-selection/) for Kubernetes 1.18+.
+- SDS for mixer? Not sure we need this or its an implementation detail
+- ***Improved*** Prometheus integration experience by adding standard Prometheus scrape annotations to proxies and the control plane workloads. This removes the need for specialized configuration to discover and consume Istio metrics. More details are availabe in the [design doc](https://docs.google.com/document/d/1TTeN4MFmh4aUYYciR4oDBTtJsxl5-T5Tu3m3mGEdSo8/edit).
```

Comment:
is it ok to link to Google docs?

Reply:
We should use https://preliminary.istio.io/docs/ops/integrations/prometheus#option-2-metrics-merging

Reply:
updated